spec-0014/index.md

```diff
@@ -0,0 +1,370 @@
+---
+title: "SPEC 14 — Interactive Documentation"
```

Comment:
Perhaps consider a more specific name than Interactive Documentation. This SPEC focuses on WebAssembly enabled interactive documentation. Binder offers a way to serve interactive documentation when the pyodide/emscriptem approach is unable to support a scientific library or an example with dependencies not supported by wasm. If leaving the title as Interactive Documentation, I think it would be helpful to include a reference to Binder as an alternative when wasm is unsuitable for an example.